fineract-provider/src/main/java/org/apache/fineract/infrastructure/configuration/domain/ConfigurationDomainServiceJpa.java

```diff
@@ -501,4 +502,19 @@ public String retrieveReportExportS3FolderName() {
         final GlobalConfigurationPropertyData property = getGlobalConfigurationPropertyData(REPORT_EXPORT_S3_FOLDER_NAME);
         return property.getStringValue();
     }
+
+    @Override
+    public String getAccrualDateConfigForCharge() {
+        String defaultValue = "due-date";
+        final GlobalConfigurationPropertyData property = getGlobalConfigurationPropertyData(CHARGE_ACCRUAL_DATE_CRITERIA);
+        if (property.isEnabled()) {
```

Comment:
I don't think we need the enabled check here (at least judging from the other global configs). It's only used for boolean values.